tf2onnx/tfonnx.py

```diff
@@ -866,6 +866,24 @@ def cast_op(ctx, node, name, args):
     return node
 
 
+def sign_op(ctx, node, name, args):
+    """Sign op."""
+    # T sign = Sign(T Input)
+    nodes = []
+    node_dtype = node.dtype
+    if node_dtype in [onnx_pb.TensorProto.COMPLEX64, onnx_pb.TensorProto.COMPLEX128]:
+        raise ValueError("dtype " + node.dtype + " is not supported in onnx for now")
+    input_tensor_type = utils.ONNX_TO_NUMPY_DTYPE[node_dtype]
```

Comment:
utils.make_sure() to make sure node_dtype is not null

Reply:
sure, btw: why don't put this check into `dtype` method?